src/main.rs

```diff
@@ -432,13 +432,14 @@ fn main() {
         };
 
         // Search for command in cache
-        if let Some(path) = cache.find_page(&command, &languages) {
-            if let Err(msg) = print_page(&path, args.flag_markdown, &config) {
-                eprintln!("{}", msg);
-                process::exit(1);
-            } else {
-                process::exit(0);
+        if let Some(paths) = cache.find_pages(&command, &languages, &config) {
+            for path in paths.iter() {
```

Comment:
This does not really tell me, that `paths.iter()` actually gives me two connected (and properly ordered) paths to a page and maybe a path, but rather seems as if multiple pages are output. This does however produce the expected output, so it is more about naming.

I suggest you call `print_page` once with the `LookupResult` and adjust this function to print everything found. To print both files in the function, you may refactor everything in it into a closure and call that on both paths contained in the `LookupResult`.

Reply:
An implication of this is also that I would opt to keep the name `find_page` instead of `find_pages`, because only one page (plus maybe a _patch_) would be found, right?

Reply:
Renamed back to `find_page` and made `iter` -> `paths`